modules/cluster-logging-deploy-clo-cli.adoc

```diff
@@ -0,0 +1,264 @@
+
+NAMESPACE                                               NAME                                         DISPLAY                  VERSION               REPLACES   PHASE
+...
+openshift-logging                                       clusterlogging.4.3.1-202002032140            Cluster Logging          4.3.1-202002032140               Succeeded
+...
+----
+
+. Create a Cluster Logging instance:
+
+.. Create an instance object YAML file (for example, `clo-instance.yaml`) for the Cluster Logging Operator:
++
+[NOTE]
+====
+This default Cluster Logging configuration should support a wide array of environments. Review the topics on tuning and
+configuring the Cluster Logging components for information on modifications you can make to your Cluster Logging cluster.
+====
++
+ifdef::openshift-dedicated[]
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "elasticsearch"
+    elasticsearch:
+      nodeCount: 3
+      storage:
+        storageClassName: "gp2"
+        size: "200Gi"
+      redundancyPolicy: "SingleRedundancy"
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      resources:
+        request:
+          memory: 8G
+  visualization:
+    type: "kibana"
+    kibana:
+      replicas: 1
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+  curation:
+    type: "curator"
+    curator:
+      schedule: "30 3 * * *"
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+  collection:
+    logs:
+      type: "fluentd"
+      fluentd: {}
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+----
+endif::[]
+
+ifdef::openshift-enterprise,openshift-origin[]
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance" <1>
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed"  <2>
+  logStore:
+    type: "elasticsearch"  <3>
+    elasticsearch:
+      nodeCount: 3 <4>
+      storage:
+        storageClassName: gp2 <5>
+        size: 200G
+      redundancyPolicy: "SingleRedundancy"
+  visualization:
+    type: "kibana"  <6>
+    kibana:
+      replicas: 1
+  curation:
+    type: "curator"  <7>
+    curator:
+      schedule: "30 3 * * *"
+  collection:
+    logs:
+      type: "fluentd"  <8>
+      fluentd: {}
+----
+<1> The name must be `instance`.
+<2> The Cluster Logging management state. In most cases, if you change the default Cluster Logging defaults, you must set this to `Unmanaged`.
+However, an unmanaged deployment does not receive updates until Cluster Logging is placed back into the `Managed` state. For more information, see *Changing cluster logging management state*.
+<3> Settings for configuring Elasticsearch. Using the CR, you can configure shard replication policy and persistent storage. For more information, see *Configuring Elasticsearch*.
```

Comment:
s/CR/Customer Resource (CR)/ 
Because it is first instance

Reply:
outdated